packages/ses/src/permits-intrinsics.js

```diff
@@ -303,32 +298,22 @@ export default function whitelistIntrinsics(
             if (typeof obj === 'function' && prop === 'prototype') {
               obj.prototype = undefined;
               if (obj.prototype === undefined) {
-                inConsoleGroup(
-                  'warn',
-                  `Tolerating undeletable ${subPath} === undefined`,
-                );
+                warn(`Tolerating undeletable ${subPath} === undefined`);
                 // eslint-disable-next-line no-continue
                 continue;
               }
             }
-            inConsoleGroup('error', `failed to delete ${subPath}`, err);
+            error(`failed to delete ${subPath}`, err);
           } else {
-            inConsoleGroup('error', `deleting ${subPath} threw`, err);
+            error(`deleting ${subPath} threw`, err);
           }
           throw err;
         }
       }
     }
   }
 
-  try {
-    // Start path with 'intrinsics' to clarify that properties are not
-    // removed from the global object by the whitelisting operation.
-    visitProperties('intrinsics', intrinsics, permitted);
-  } finally {
-    if (groupStarted) {
-      // eslint-disable-next-line @endo/no-polymorphic-call
-      console.groupEnd();
-    }
-  }
+  // Start path with 'intrinsics' to clarify that properties are not
+  // removed from the global object by the whitelisting operation.
+  visitProperties('intrinsics', intrinsics, permitted);
```

Comment:
Why do we no longer need the try/finally? Or did we already not need it?

Reply:
The `try/finally` moved into a utility function in `console.js` (to be renamed `reporting.js`, since it now gets used for this and separately enablements warnings, under a different group label.

Since try/finally deopts the surrounding function, this has the added benefit of deopting a smaller function.